src/input/type_output.hh

```diff
@@ -11,6 +11,7 @@
 
 #include "input/type_base.hh"
 #include "input/type_record.hh"
+#include "input/type_generic.hh"
 #include <boost/regex.hpp>
```

Comment:
RewriteRule is used only in escape_description and is mere pair. So, we can remove class RewriteRule and use 
static vector of pairs in the method OutputJSONMachine::escape_description.